### ISSUE TITLE
Draft watermark appears even though "draft = no"

### ISSUE THREAD

**Issue by amw290** [Microbiology/PandocAcademicCoverLetter#2]
Hi there,
I'm having trouble with my PDF without a watermark. This heading produces a watermark.
---
size: 11pt
draft: no
head:
    address: '616 Serra Street'
    city: Stanford
    state: CA
    zip: 94040
    phone: '\(312\) 479-2182'
    email: 'weichselbraun@stanford.edu'
    dept: "Center for International Security and Cooperation"
sig:
 include: yes
 sign: yes
 name: 'Anna Weichselbraun, PhD'
 post: Lecturer
 secondpost:
---

**Comment by amw290** [comment]
And when I %% out the watermark and delete the "draft" line from the markdown file I get some other error about the Image file...

**Comment by amw290** [comment]
this error:
Error producing PDF.
! Undefined control sequence.
<argument> \includegraphics
                            [height=1in]{./Stanford.pdf}
l.62 ...or International Security and Cooperation}

**Comment by Microbiology** [comment]
My first guess would be that it has something to do with that final error you mentioned (the watermark itself is at line 57 - 60 in `cltemplate.tex`. Hopefully this gets clearer after #1 is fixed.

**Comment by amw290** [comment]
Yes, I think so too. I tried commenting out lines 57-60 and would then run
into the other error.

**Comment by mgiugliano** [comment]
I am on macOs, installed pandoc (2.3) with brew, and still have the same bug.

From http://pandoc.org/MANUAL.html (search for "Using Variables in templates") I understand that boolean variables as YAML should take the values 'true' or 'false', rather than 'yes' or 'no'.

In fact, commenting line 57-60 in cltemplate.tex (or deleting them) gives the same error if the YAML value is set as "draft: false". 

I do not understand why - under these circumstances - one gets the error 

! Undefined control sequence.
<argument> \includegraphics

It is as if the (LaTeX) package to include graphics is not included explicitly but it gets "inherited", so to speak, by the watermark package inclusion....

**Comment by mgiugliano** [comment]
Adding the line 

```\usepackage{graphicx}       % Needed to insert images into the document```

and using 'true' and 'false', instead of 'yes' and 'no', works!
